api/resolvers/chainFee.js

```diff
@@ -0,0 +1,37 @@
+import lndService from 'ln-service'
+import lnd from '../lnd'
+
+const cache = new Map()
+const expiresIn = 1000 * 30 // 30 seconds in milliseconds
+
+async function fetchChainFeeRate () {
+  let chainFee = 0
+  try {
+    const fee = await lndService.getChainFeeRate({ lnd })
```

Comment:
I believe this is only the estimate since it says "estimate" here:

> **getChainFeeRate**
> Get chain fee rate estimate

-- https://www.npmjs.com/package/ln-service#getchainfeerate

Mentioning this, because when I first saw the ticket, I thought if I am going to work on this, I am going to use the median fee rate of the recent block if possible.

But not sure if possible using LND RPC calls.

So I think this is also fine :)